test/common.js

```diff
@@ -1,9 +1,28 @@
 'use strict';
+
+process.env.NODE_ENV = 'test';
+
 require('dotenv').config();
 const _ = require('lodash');
 const nock = require('nock');
 const apiServer = require('./fixtures/api');
 const fixtures = require('./fixtures');
+const config = require('../config');
+
+function clearDB() {
+  const tables = [
+    'oauth_credentials',
+    'users',
+  ];
+  let deferred = config.bookshelf.knex.migrate.latest();
+
+  for (const tableName of tables) {
+    // eslint-disable-next-line no-loop-func
+    deferred = deferred.then(() => config.bookshelf.knex(tableName).select().del());
```

Comment:
Could we call here something like `config.bookshelf.knex.migrate.<to null state>();` instead of manual table deletion? It will test our down migrations also.